app/components/views/TicketsPage/Header.js

```diff
@@ -1,141 +1,94 @@
-          {purchaseTicketsSuccess}
-        </div>
-      ) : null,
-      revokeTicketsError ? (
-        <div key="revokeTicketsError" className="stakepool-view-notification-error">
-          <div className="stakepool-content-nest-address-delete-icon" onClick={onClearRevokeTicketsError}/>
-          {revokeTicketsError}
-        </div>
-      ) : null,
-      revokeTicketsSuccess ? (
-        <div key="revokeTicketsSuccess" className="stakepool-view-notification-success">
-          <div className="stakepool-content-nest-address-delete-icon" onClick={onClearRevokeTicketsSuccess}/>
-          {revokeTicketsSuccess}
-        </div>
-      ) : null,
-      startAutoBuyerSuccess ? (
-        <div key="startAutoBuyerSuccess" className="stakepool-view-notification-success">
-          <div className="stakepool-content-nest-address-delete-icon" onClick={onClearStartAutoBuyerSuccess}/>
-          {startAutoBuyerSuccess}
-        </div>
-      ) : null,
-      stopAutoBuyerSuccess ? (
-        <div key="stopAutoBuyerSuccess" className="stakepool-view-notification-success">
-          <div className="stakepool-content-nest-address-delete-icon" onClick={onClearStopAutoBuyerSuccess}/>
-          {stopAutoBuyerSuccess}
-        </div>
-      ) : null,
-      startAutoBuyerError ? (
-        <div key="startAutoBuyerError" className="stakepool-view-notification-error">
-          <div className="stakepool-content-nest-address-delete-icon" onClick={onClearStartAutoBuyerError}/>
-          {startAutoBuyerError}
-        </div>
-      ) : null,
-      stopAutoBuyerError ? (
-        <div key="stopAutoBuyerError" className="stakepool-view-notification-error">
-          <div className="stakepool-content-nest-address-delete-icon" onClick={onClearStopAutoBuyerError}/>
-          {stopAutoBuyerError}
-        </div>
-      ) : null,
-      importScriptError ? (
-        <div key="importScriptError" className="stakepool-view-notification-error">
-          <div className="stakepool-content-nest-address-delete-icon" onClick={onClearImportScriptError}/>
-          {importScriptError}
+const opts = { stiffness: 150, damping: 20 };
+
+const messages = defineMessages({
+  "tickets.title":               { id: "tickets.title",               defaultMessage: "Tickets" },
+  "tickets.description":         { id: "tickets.description",         defaultMessage: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua." },
+  "tickets.tab.purchase":        { id: "tickets.tab.purchase",        defaultMessage: "Purchase Tickets" },
+  "tickets.tab.mytickets":       { id: "tickets.tab.mytickets",       defaultMessage: "My Tickets" },
+  "tickets.tab.governance":      { id: "tickets.tab.governance",      defaultMessage: "Governance" },
+  "tickets.tab.statistics":      { id: "tickets.tab.statistics",       defaultMessage: "Statistics" },
+});
+
+@autobind
+class Header extends React.Component {
```

Comment:
This thing could realistically be pulled out into it's own component instead of being defined in both tickets and transactions.  Maybe that should be a separate PR?

Reply:
Yah that was my thought as well.  We can do in a seperate PR